@ndlib/gatsby-theme-marble/src/components/App/FormElements/TextArea/index.js

```diff
@@ -14,6 +14,7 @@ const TextArea = ({ id, label, defaultValue = '', disabled = false }) => {
         className={style.editTextArea}
         defaultValue={defaultValue}
         disabled={disabled}
+        onChange={(event) => onChange(event)}
```

Comment:
You could just do `onChange={onChange}` instead of creating an anonymous function that just does a passthrough.